docker/start-notebook.sh

```diff
@@ -4,4 +4,11 @@
 
 set -e
 
+MYSQL_CMD="mysql -h $MYSQL_HOST -u root -p$MYSQL_ROOT_PASSWORD"
+
+until $MYSQL_CMD -e ";"; do
+  >&2 echo "MySQL is unavailable - sleeping"
+  sleep 1
+done
+
```

Comment:
That _was_ waiting so we could add the user, but since we moved that to the provisioning container, we don't need it anymore.